tests/it.xsemantics.dsl.tests/tests_input_files/ecore_errspecification_test.xsemantics

```diff
@@ -16,6 +16,19 @@ import org.eclipse.emf.ecore.EObject
 
 system it.xsemantics.test.errspecification.ecore.TypeSystem
 
+copyright
+"/*******************************************************************************
+ * Copyright (c) 2013-2017 Lorenzo Bettini.
```

Comment:
@LorenzoBettini @mor-n4  This is a valid usage of the copyright variable: The comment syntax must be part of the string now.